src/areachart/AreaChart.jsx

```diff
@@ -26,7 +26,9 @@ module.exports = React.createClass({
       interpolate: false,
       interpolationType: null,
       className: 'rd3-areachart',
-      hoverAnimation: true
+      hoverAnimation: true,
+			labelsAccessor: d => d.name,
```

Comment:
Please follow the indentation. If you are using Sublime Text you'll need to turn tab into space